imports/stylesheets/mixins.import.styl

```diff
@@ -80,3 +80,16 @@ gradientTopAndBottom()
   &::after
     background linear-gradient(to bottom, alpha(white, 0%), white)
     bottom 0
+
+gradientBottom()
+  &::after
+    content ''
+    height 20px
+    position absolute
+    left 0
+    right 0
+    z-index 10
+  &::after
+    background linear-gradient(to bottom, alpha(white, 0%), white)
+    bottom 0
```

Comment:
There's a class called `gradient-bottom` [here](https://github.com/ecohealthalliance/eidr-connect/blob/343dc99aacea92a7f0761ccf76d523a5983e8891/imports/stylesheets/utils.import.styl#L35-L46). Maybe we could now call the new mixin in that class and pass in the height as a variable (with a default) for flexibility.

Also, we can remove that second `&::after` selector